File: 6.0001/ps0/solution.py
import math
import numpy

x = int(input("Enter number x: "))
y = int(input("Enter number y: "))

# Math.pow returns a float and ** keep the original types
print("X**y = ", math.pow(x, y))

# Is numpy.log2 more accurate than math.log?
# Or does it just automatically round which makes it more user-friendly
print("log(x) = ", numpy.log2(x))

File: 6.0001/ps0/solution.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x, y float64

	fmt.Print("Enter number x: ")
	fmt.Scan(&x)

	fmt.Print("Enter number y: ")
	fmt.Scan(&y)

	fmt.Println("X**y is", math.Pow(x, y))

	fmt.Println("log(x) is", math.Log2(x))
}
